### DISCUSSION TITLE
Alguém aqui tem alguma experiência com mentores?

### DISCUSSION THREAD

**Discussion by JGcarv** [frontendbr/forum#108] [Discussão]
Lendo algumas discussões sobre carreira, comecei a notar que mentores são relativamente comuns lá fora. Alguém aqui tem algum tipo de experiência parecida?

**Comment by willycamargo** [comment]
Qual a definição de "mentor" para você?

Existem algumas pessoas as quais me baseio e confio para tirar minhas dúvidas, assim como existem pessoas que confiam em mim para vir tirar as dúvidas.

Não sei se isso pode ser considerado uma relação de "mentor <-> aprendiz", mas pra mim é algo comum.

**Comment by felipefialho** [comment]
Como seria exatamente essa experiencia @JGcarv?

**Comment by JGcarv** [comment]
@willycamargo @LFeh Eu entendo mentores como uma espécie de professor particular, só que ao invés de ensinar de fato, ele apenas indica o caminho. E não acho que se restrinja apenas a ensinamentos técnicos, mas também sobre a vida profissional e questões relacionadas. Talvez seja um conceito parecido de ''coach".

Mas, com toda honestidade, não tenho muito uma opinião formada sobre o assunto, talvez por isso gostaria de ouvir outras opiniões ;)

**Comment by felipefialho** [comment]
É que realmente não conheço esse conceito.

Esses professores são pagos? Ou é uma coisa mais de comunidade?

**Comment by icaromh** [comment]
Eu associo muito mentoria com lance de empreendedorismo e startup. Todo evento de aceleração tem mentores lá pra ajudar. Mas é o único lugar que vi sendo aplicado aqui no Brasil.

**Comment by baltazarparra** [comment]
Pensando pelo lado comunitário, seria massa, se Dev's mais experientes, "adotassem" grupos limitados de novos desenvolvedores (Pagos, ou selecionados de outra forma), para ir acompanhando a evolução e tal, algo próximo a esse conceito de Coach/Mentor.

Não sei até que ponto seria viável, mas creio que seria de grande vália pra comunidade como um todo.

**Comment by sanderweb** [comment]
Muito interessante o assunto, tem link para conferir????
E porque não nos reunimos e montamos algo assim por aqui????

**Comment by felipefialho** [comment]
@baltazarparra @sanderweb A ideia é interessante, porque não começam a rascunhar? https://github.com/frontendbr/open-source =)

**Comment by baltazarparra** [comment]
Podemos montar um grupo de discussão, mas seria sensato se alguém que pudesse assumir o status de "mentor" se junta-se a nós, para ficar mais fácil trazer isso pra realidade.

Você se juntaria ao grupo de discussão @LFeh ?

**Comment by felipefialho** [comment]
@baltazarparra Acho legal a ideia de grupo de discussão. Mas...

Eles precisam ser escaláveis e públicos, justamente para dar a possibilidade de pesquisa e acesso para pessoas de "fora desse grupo". Alias esse foi o principal motivo de portar discussões do Facebook e outras redes pra cá.

Portanto o ideal é pensar em soluções que seguem esse conceito ;)

**Comment by baltazarparra** [comment]
Perfeito, podemos montar um brainstorm em uma issue? Ou tem alguma outra sugestão?

**Comment by felipefialho** [comment]
Sugiro abrir uma [issue no repositório de open-source](https://github.com/frontendbr/open-source) e agitar uma galera para discutir isso.

**Comment by baltazarparra** [comment]
Criado @LFeh 

[Mentoria no Front](https://github.com/frontendbr/open-source/issues/5)

**Comment by felipefialho** [comment]
Perfeito. Divulga a issue em Facebook, Slack e afins pra ver se tem mais gente disposta a ajudar ;)

A ideia é bem bacana. 

O que achou @JGcarv ?

**Comment by baltazarparra** [comment]
@LFeh não uso facebook, mas já divulguei pra uma galera no slack, se alguém puder passar isso lá dentro do facebook seria massa.

**Comment by JGcarv** [comment]
@LFeh Vim aqui perguntar porque estava justamente considerando encontrar alguém assim. Fico feliz que isso tenha despertado  interesse de outras pessoas!

**Comment by karlprieb** [comment]
@baltazarparra Parabéns por trazer isso para nossa comunidade :) Muito legal a ideia!

**Comment by sanderweb** [comment]
Olá a todos! Tem muita gente fera e experiente aqui, gostaria de propor uma coisa para os novatos e quem esta começando, escolher um curso gratuito web "x" montar uma turma para fazer juto e debater sobre com repositório aqui no Guit? Será que tem algum novato aqui ou posso trazer a ideia pra aqui??? Eu sei que o pessoal mais experiente não vai querer participar mais o que acham?

**Comment by woliveiras** [comment]
Taca pau aqui @sanderweb: http://github.com/training-center/sobre

**Comment by sanderweb** [comment]
Show muito obrigado @woliveiras Vlw!
